rusqlite_migration/src/lib.rs

```diff
@@ -658,6 +658,19 @@ impl<'m> Migrations<'m> {
     /// ```
     /// If rusqlite `extra_check` feature is enabled, any migration that returns a value will error
     /// and no further migrations will be applied.
+    ///
+    /// # Transaction Behavior
+    ///
+    /// Since rusqlite 0.33, a [default transaction behavior][default_behavior] can be set. For
+    /// now, when applying migrations, this setting will be respected.
+    ///
+    /// Please note that future minor versions of rusqlite_migration might decide to ignore the
+    /// setting and to instead use any transaction behavior deemed most appropriate.  You can read
+    /// more in the [corresponding page of the SQLite documentation][sqlite_doc].
```

Comment:
I’m relunctant to tie our hands if it turns out that one of the modes causes issues in some circumstances. I don’t think it should but it’s hard to predict, I think.

Any objections?

Reply:
Maybe we can learn more during the beta actually.